### PR TITLE
chercolvin's First Goodie

### PR DESCRIPTION
**Pull request by chercolvin**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
It displays the results of my first Goodie: "chercolvin is awesome..."

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It just solves that I can submit a Goodie IA and a pull request.

**What is the data source for your Instant Answer? (Provide a link if possible)**
duckduckgo

**Why did you choose this data source?**
It's in the tutorial.

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**
duckduckhack chercolvin

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
special, special interest, geek

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
It's part of the tutorial.

**Which existing Instant Answers will this one supercede/overlap with?**
None that I know of.

**Are you having any problems? Do you need our help with anything?**
No.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Through the DuckDuckGo twitter feed.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [x] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```
goodie snip

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/chercolvin.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+name "IsAwesome chercolvin";
+description "My first Goodie, it lets the world know that chercolvin is awesome";
+primary_example_queries "duckduckhack chercolvin";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/chercolvin.pm";
+attribution github => ["chercolvin", "Cheryl Colvin"],
+            twitter => "chercolvin";
+
+# Triggers
```
Comment (javathunderman) [inline lib/DDG/Goodie/IsAwesome/chercolvin.pm]: You can remove this and the # Handle Statement line.

**Comment by javathunderman** [comment]
@chercolvin Looks good! Left one little note for you. 

cc @moollaza @mintsoft

**Comment by mintsoft** [comment]
Apologies for the delay @chercolvin Congratulations on your first goodie! :cake:

**Comment by moollaza** [comment]
@chercolvin congrats! It's live: https://duckduckgo.com/?q=duckduckhack+chercolvin&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**